app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
There is no need to pass `" "` to split, it is a default option, remove it
```suggestion
    command_list = command.split()
```